client/less/classic-modal.less

```diff
@@ -1,34 +0,0 @@
-.challenge-success-modal {
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
-  height: 50vh;
-
-  .modal-header {
```

Comment:
Somewhere along the way I think we lost the background colour to the `.modal-header`, it must have been inherited. 

So here will need to be 
```css
.modal-header {
  background-color: @brand-primary;
  color: #fff;
  margin-bottom: 0;
}
```